config/routes.rb

```diff
@@ -24,4 +24,19 @@
   if Rails.env.development? || Rails.env.dev?
     get "/sandbox", to: "sandbox#index"
   end
+
+  resources :challenges do
+    resource :manage_evaluators, only: [] do
+      collection do
+        get :index
+        post :create
+        delete :destroy
+      end
+    end
```

Comment:
I believe this is the standard way for routing:
```suggestion
    resources :manage_evaluators, only: [:index, :create, :destroy]
```